hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/DelegationTokenRenewer.java

```diff
@@ -107,7 +107,7 @@ public boolean equals(final Object that) {
      * Set a new time for the renewal.
      * It can only be called when the action is not in the queue or any
      * collection because the hashCode may change
-     * @param newTime the new time
+     * @param delay the renew cycle
```

Comment:
Shouldn't this be.
`the renewal time` 
rather than `the renew cycle`

Reply:
Thanks @ayushtkn for good suggestion, it's more reasonable.
Updated this in latest PR.